### ISSUE TITLE
DNN 9.5.0 breaks Catalog, OC 4.4.0

### ISSUE THREAD

**Issue by TopCat6** [sachatrauwaen/OpenContent#115]
Just upgraded my test site to DNN 9.5.0 and the Catalog template doesn't allow me to edit anymore. The Edit and the Add screens come up empty.  I used Switch Template to downloaded a new version through the interface and named the new directory a new name. Still same problem.

Anybody else having this issue?

**Comment by b-creative** [comment]
I've got a DNN 9.5.0 and OC 4.4.0 setup also... no problems so far.
Do you see errors in the console?

**Comment by sachatrauwaen** [comment]
Do you have any javascript errors in the browserconsole ?

**Comment by TopCat6** [comment]
Thanks for responding.  Yes, I see a 'too much recursion' error when I do an Add or Edit.
If I do an Edit:

```
JQMIGRATE: Migrate is installed, version 3.1.0       jquery-migrate.js:2:696
too much recursion                                   jquery.js:2:32085
    RegExpLocalReplaceOptShort self-hosted:2438
    Symbol.replace self-hosted:2098
    replace self-hosted:2985
    jQuery 125
        V
        set
        De
        setup
        add
        add
        De
        setup
```
If I do an Add, I get this:

```
JQMIGRATE: Migrate is installed, version 3.1.0        jquery-migrate.js:2:696
too much recursion                                    jquery.js:2:40656
    RegExpBuiltinExec self-hosted:2721
    exec self-hosted:2683
    jQuery 126
        add
        add
        De
        setup
        add
        add
        De
        setup
```
I have tried deleting the module, removing the Catalog template folder, then adding a new module and selecting Catalog from github, but still have same symptoms.

**Comment by b-creative** [comment]
Have you tried using the DNN Xcillion Skin? On your page?

**Comment by TopCat6** [comment]
It is already using the Xcillion skin.

**Comment by TopCat6** [comment]
I just restored the backup for the DNN 9.4.4 directory tree, done just before upgrade to DNN 9.5.0.  No errors on console and both Edit and Add work properly.
Hmmm...

**Comment by TopCat6** [comment]
Just updated DNN to 9.6.0 and this problem does not occur.

**Comment by schotman** [comment]
This is a jquery 3.4.1 issue, solved with jquery 3.5 in 9.6 (better use 9.6.1 / jquery 3.5.1)
